dev/src/bulk-writer.ts

```diff
@@ -434,10 +455,27 @@ export class BulkWriter {
     data: T
   ): Promise<WriteResult> {
     this.verifyNotClosed();
-    const bulkCommitBatch = this.getEligibleBatch();
-    const resultPromise = bulkCommitBatch.create(documentRef, data);
+    const op = this._create(documentRef, data);
```

Comment:
Instead of having a helper function for each call that uses a lambda internally, can you just call `wrapRetry` and pass `bulkCommitBatch.create` directly?